taichi/ir/ir_builder.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include "taichi/ir/ir.h"
+
+TLANG_NAMESPACE_BEGIN
+
+class IRBuilder {
+ public:
+  struct InsertPoint {
```

Comment:
nit: does this need to be public?

Reply:
I'm not sure if we need to modify the insertion point... I think it's fine to make it private now.